### PR TITLE
Update Documenter version

### PR DESCRIPTION
**Pull request by schillic**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 [compat]
 CDDLib = "0.6, 0.7"
 Distributions = "0.19, 0.20, 0.21, 0.22, 0.23, 0.24, 0.25"
-Documenter = "0.23, 0.24, 0.25, 0.26"
+Documenter = "0.23, 0.24, 0.25, 0.26, 0.27"
```
Comment (mforets) [inline Project.toml:24]: should we keep all those previous versions?

**Review by mforets — APPROVED** [review]

**Comment by mforets** [comment]
> ERROR: LoadError: AssertionError: length(exs) == 2 * length(results)

seems to be Documenter issue? perhaps we should report in their issue tracker.

**Comment by schillic** [comment]
I am debugging it.
```julia
# length == 4
exs = Any[
:(#= set_operations.md:1 =#),
:(println(is_intersection_empty(BI, H))),
:(#= set_operations.md:2 =#),
:(w1 = (is_intersection_empty(BI, H, true))[2])
]

# length == 3
results = Any[(
(:(println(is_intersection_empty(BI, H))), "println(is_intersection_empty(BI, H))\n"),
(:(w1 = (is_intersection_empty(BI, H, true))[2]), "w1 = is_intersection_empty(BI, H, true)[2]\n"),
(nothing, "# none of the other combinations are supported yet\n# is_intersection_empty(B1, B2)\n# is_intersection_empty(B1, BI)\n# is_intersection_empty(B1, H)\n# w2 = is_intersection_empty(B1, H, true)[2]\n# is_intersection_empty(B2, BI)\n# is_intersection_empty(B2, H)\n")
]
```
The last element is probably wrong.

File: Project.toml (hunk 1 above)
Comment (schillic) [reply Project.toml:24]: I think it does not hurt. Sometimes other packages force a downgrade.

**Review by mforets — APPROVED** [review]

**Comment by mforets** [comment]
for future reference, the deleted comments state some disjointness check operations which are not supported yet:

```
# none of the other combinations are supported yet
# is_intersection_empty(B1, B2)
# is_intersection_empty(B1, BI)
# is_intersection_empty(B1, H)
# w2 = is_intersection_empty(B1, H, true)[2]
# is_intersection_empty(B2, BI)
# is_intersection_empty(B2, H)
```

**Comment by schillic** [comment]
Yes, but some of them work now. The `Ball2` combinations are missing.

**Pull request merged**